src/caffe/layers/cudnn_conv_layer.cpp

```diff
@@ -170,26 +172,19 @@ void CuDNNConvolutionLayer<Dtype>::Reshape(
   if (use_algo_seeker_) {
     size_t workspace_limit_bytes, total_memory;
     GPUMemory::GetInfo(&workspace_limit_bytes, &total_memory);
-    // FindEx: A workspace of size workspace_bytes is allocated for FindEx.
+    // FindEx: A workspace of size WORKSPACE_SIZE is allocated for FindEx.
```

Comment:
I think the size of workspace used for FindEx and Get is still `workspace_bytes`; `WORKSPACE_SIZE` is set after FindEx or Get by `UpdateWorkspaceDemand`.